tests/unit/AppContext/Application/CommandHandler/Repository/RegisterTest.php

```diff
@@ -77,8 +77,8 @@ public function testItReturnsTheRepositoryIfItAlreadyExists()
         $repository = $this->createMock(Entity\Repository::class);
 
         $this->repositoriesRepo->expects($this->once())
-            ->method('find')
-            ->with('some identifier')
+            ->method('findByIdentifier')
+            ->with(Entity\Repository::TYPE_GITHUB, 'some identifier')
             ->willReturn($repository);
 
         $command = new Command\Repository\Register($user, Entity\Repository::TYPE_GITHUB, 'some identifier', 'some name', 'shared secret');
```

Comment:
Parameter #1 $owner of class Regis\AppContext\Application\Command\Repository\Register constructor expects Regis\Kernel\User, PHPUnit_Framework_MockObject_MockObject given.